### PR TITLE
igt: get Chamelium IP address from shell environment variable

### PR DESCRIPTION
**Pull request by arthur-she**
Now we already have LAVA support allowing test shell to query device
dictionary. So the DUT can get the paired Chamelium IP address from
shell environment variable ${CHAMELIUM_STATIC_IP}

Signed-off-by: Arthur She <arthur.she@linaro.org>

### CODE REVIEW

**Comment by arthur-she** [comment]
Test job
https://validation.linaro.org/scheduler/job/1935848

File: automated/linux/igt/igt-chamelium-test.yaml [hunk 1]
```diff
@@ -31,6 +30,8 @@ run:
         - cd ./automated/linux/igt
         - git config --global http.sslverify false
         - if [ -n "${TEST_LIST}" ]; then TL="-t ${TEST_LIST}"; fi
+        # ${CHAMELIUM_STATIC_IP} is from LAVA device dictionary
+        - CHAMELIUM_IP=`echo ${CHAMELIUM_STATIC_IP}`
```
Comment (mwasilew) [inline automated/linux/igt/igt-chamelium-test.yaml]: what is the reason for using 'echo'? Wouldn't simple assignment work? Also, maybe we should simply change the name of the variable in device dictionaries to match the script?

**Comment by arthur-she** [comment]
The variable name has been changed from CHAMELIUM_STATIC_IP to CHAMELIUM_IP.
So I revised the pull request accordingly.

**Comment by mwasilew** [comment]
I wonder which variable is more important - from device dict or from test definition. I'll merge it and let's see if it works.

**Pull request merged**